src/components/scenes/SettingsScene.tsx

```diff
@@ -175,15 +176,17 @@ export class SettingsSceneComponent extends React.Component<Props, State> {
         warning
         onSubmit={async text => {
           if (text !== username) return s.strings.delete_account_verification_error
-
           await this.props.account.deleteRemoteAccount()
           await this.props.logoutRequest()
           await this.props.context.deleteLocalAccount(username)
-          showToast(sprintf(s.strings.delete_account_feedback, username))
+
           return true
         }}
       />
-    ))
+    )).then(() => {
+      const message = sprintf(s.strings.delete_account_feedback, username)
+      Airship.show(bridge => <TextDropdown bridge={bridge} message={message} />)
```

Comment:
The way we launch the dropdown looks great. However, I am worried that it will show even if the user cancels the modal, since we aren't checking the modal's return value.

Reply:
I have moved the `.then(... <TextDropdown>` after `this.props.context.deleteLocalAccount(username)`. This is more nested but guarantees the modal only shows up after all deletion logic has succeeded.